src/main/java/ch/wisv/areafiftylan/users/controller/CurrentUserRestController.java

```diff
@@ -89,8 +90,17 @@
     @PostMapping("/password")
     public ResponseEntity<?> changeCurrentUserPassword(@AuthenticationPrincipal User user,
                                                        @RequestBody @Validated PasswordChangeDTO passwordChangeDTO) {
-        userService.changePassword(user.getId(),
-                passwordChangeDTO.getOldPassword(), passwordChangeDTO.getNewPassword());
+        try {
+            userService.changePassword(user.getId(),
+                    passwordChangeDTO.getOldPassword(), passwordChangeDTO.getNewPassword());
+        } catch (Exception e) {
```

Comment:
You don't catch any other errors than the two listed, which would then be silently ignored. Maybe better to add a simple case with returns a 500 error?

Reply:
I looked at the errors it could throw, in principle it only throws the two listed, but I agree with adding the last case